external/.clang-format

```diff
@@ -0,0 +1,7 @@
+# Copyright 2024 The Khronos Group Inc.
+# SPDX-License-Identifier: Apache-2.0
+---
+# Disable clang-format in this directory
+DisableFormat: true
+SortIncludes: false
```

Comment:
Why is this needed when formatting is disabled?

Reply:
It's probably not strictly necessary.

Reply:
This is the convention used in other Khronos repositories, but it should have no effects (at least on clang-format versions I've used).